src/compiler/checker.ts

```diff
@@ -24019,6 +24022,7 @@ namespace ts {
                 case ExternalEmitHelpers.AsyncDelegator: return "__asyncDelegator";
                 case ExternalEmitHelpers.AsyncValues: return "__asyncValues";
                 case ExternalEmitHelpers.ExportStar: return "__exportStar";
+                case ExternalEmitHelpers.MakeTemplateObject: return "__makeTemplateObject";

```

Comment:
Not added in this PR, but all of the strings here are duplicated elsewhere in the emitter -- could we move this function to there?

Reply:
https://github.com/Microsoft/TypeScript/issues/18863